modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache

```diff
@@ -150,15 +150,16 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
         return mapper;
     }
 
-    protected void init() {
+    protected static Map<String, Authentication> buildAuthentications() {{=<% %>=}}{<%={{ }}=%>{{#authMethods.0}}
```

Comment:
It seems to be a breaking change btw. Probably not master should be targeted

Reply:
thanks for the PR.

what about adding an option to let people choose which JDK version to support in the Java client/server?

and then you can use {{#jdk21}} ... {{/jdk21}} to include the change above only if users want a JDK21 compatible output?